src/dmd/dmodule.d

```diff
@@ -422,6 +422,7 @@ extern (C++) final class Module : Package
     bool isHdrFile;             // if it is a header (.di) file
     bool isDocFile;             // if it is a documentation input file, not D source
     bool isPackageFile;         // if it is a package.d
+    Package pkg;                // if isPackageFile is true, the wrapping Package
```

Comment:
What is a "wrapping Package" ?

Reply:
It's the Package that contains package.d and other submodules, the one inserted to the symbol table.
I took the name from a comment in this file:

> 2. Otherwise, 'package.d' wrapped by 'Package' is inserted to the internal tree in here.

Reply:
I don't understand it there, either. Please add your explanation to the code comment.

Reply:
Do you mean adding this:
> It's the Package that contains package.d and other submodules, the one inserted to the symbol table.
Or a more detailed explanation?

Reply:
Comment improved a little and I've added my full phrase to the block explaining the complete "package.d" insertion process .